bundles/org.openhab.core/src/main/java/org/openhab/core/internal/items/ItemRegistryImpl.java

```diff
@@ -469,6 +485,22 @@ public void unsetCommandDescriptionService(CommandDescriptionService commandDesc
         }
     }
 
+    @Reference(cardinality = ReferenceCardinality.OPTIONAL, policy = ReferencePolicy.DYNAMIC)
```

Comment:
Wouldn't it make sense to make this a mandatory service and use constructor injection? It is a default and IMO should always be provided.

Reply:
You're right.